[Discover] show temporary and managed labels when necessary in the data view picker

PhilippeOberti opened the pull request:
## Summary

This PR makes a tiny change to the DataView picker shown in Discover and soon on all the page throughout Security Solution:
- We are currently showing a small icon only for selected data views that are `adhoc` (temporary). This PR replaces the icon with a `Temporary` text label, and also adds the `Managed` text label when appropriate.
- We also currently show on the `Managed` text label in the dropdown, even if the data view is also adhoc. The PR makes sure that we display both `Temporary` and `Managed` labels when necessary

> [!WARNING]
> UIUX has not yet given the correct icon, nor has approved showing 2 icons for data views that are both `adhoc` and `managed`. To be confirmed soon!

For data views that are neither `managed` nor `adhoc`, no visible changes:
| Before  | After |
| ------------- | ------------- |
| <img width="312" height="65" alt="Screenshot 2025-09-23 at 3 46 15 PM" src="https://github.com/user-attachments/assets/84667236-edf5-45d6-909f-c63ebf94f22d" /> | <img width="312" height="65" alt="Screenshot 2025-09-23 at 3 46 15 PM" src="https://github.com/user-attachments/assets/403338fa-553c-4aa3-9605-c942a0687f2f" /> |

For data views that are `managed` only, we now show the label:
| Before  | After |
| ------------- | ------------- |
| <img width="297" height="66" alt="Screenshot 2025-09-23 at 3 46 57 PM" src="https://github.com/user-attachments/assets/4c95eba9-1a1e-4fa0-b31d-cd9b3b476524" /> | <img width="375" height="48" alt="Screenshot 2025-09-29 at 11 45 06 AM" src="https://github.com/user-attachments/assets/bddedb12-00f2-4b06-8571-3d5bbf41f27b" /> |

For data views that are `adhoc` only, we switched the icon for the label:
| Before  | After |
| ------------- | ------------- |
| <img width="210" height="64" alt="Screenshot 2025-09-23 at 3 47 29 PM" src="https://github.com/user-attachments/assets/195df744-02de-44b3-83d3-052ea51f4827" /> | <img width="269" height="44" alt="Screenshot 2025-09-29 at 11 46 32 AM" src="https://github.com/user-attachments/assets/e1a13e08-f520-4354-8a29-8fae099c5e20" /> |

For data views that are both `managed` and `adhoc` only, we show 2 icons:
| Before  | After |
| ------------- | ------------- |
| <img width="336" height="62" alt="Screenshot 2025-09-23 at 3 46 44 PM" src="https://github.com/user-attachments/assets/08c1ee3a-bdc6-45ac-af88-6dd830e8db72" /> | <img width="472" height="42" alt="Screenshot 2025-09-29 at 11 47 08 AM" src="https://github.com/user-attachments/assets/78cc7721-b41c-4e06-9a8a-e4b6d1af82f5" /> |


Now that both labels can be rendered in the dropwdpwn, we need a little bit more width space to ensure enough space for the data view name to be rendered.  This PR increases the width of the overall popover by 100px (as the Temporary label is around 60px wide and we have some padding...).
| Before  | After |
| ------------- | ------------- |
| <img width="368" height="390" alt="Screenshot 2025-09-29 at 11 50 53 AM" src="https://github.com/user-attachments/assets/46551828-0309-4214-96d8-34f373d53648" /> | <img width="466" height="394" alt="Screenshot 2025-09-29 at 11 48 01 AM" src="https://github.com/user-attachments/assets/13d8d0b5-8599-429e-86af-8e4f698ffc75" /> |

> [!NOTE]
> If this is approved by the @elastic/kibana-presentation team, I'll write some unit tests.

### Checklist

- [x] The PR  description includes the appropriate Release Notes section, and the correct `release_note:*` label is applied per the [guidelines](https://www.elastic.co/guide/en/kibana/master/contributing.html#kibana-release-notes-process)
- [x] Review the [backport guidelines](https://docs.google.com/document/d/1VyN5k91e5OVumlc0Gb9RPa3h1ewuPE705nRtioPiTvY/edit?usp=sharing) and apply applicable `backport:*` labels.

https://github.com/elastic/kibana/issues/235731

PhilippeOberti committed:
[Discover] show Temporary and Managed labels instead of the icon  in the data view picker button, and show both labels when necessary in the dropdown

l-suarez commented:
Thanks for all the context @PhilippeOberti

It looks like we added that ad-hoc icon more than 3 years ago as a solution to inform users about the type of data view. We already tell users if it's temporary, managed or both when we expand the dropdown to select a data view, and we do so adding a label component. I'd recommend doing the same for when it's selected instead of adding icons. This way is clearer, it'll make the select component longer, but it will be clearer. cc/ @ninoslavmiskovic 

<img width="454" height="398" alt="image" src="https://github.com/user-attachments/assets/4e88b8a2-f3f2-4eaa-b4ff-48a2d1126041" />

### Proposed solution
<img width="717" height="261" alt="image" src="https://github.com/user-attachments/assets/6c712cc3-c35e-4db8-abcf-b79775417735" />

l-suarez commented:
> > Thanks for all the context @PhilippeOberti
> > It looks like we added that ad-hoc icon more than 3 years ago as a solution to inform users about the type of data view. We already tell users if it's temporary, managed or both when we expand the dropdown to select a data view, and we do so adding a label component. I'd recommend doing the same for when it's selected instead of adding icons. This way is clearer, it'll make the select component longer, but it will be clearer. cc/ @ninoslavmiskovic
> 
> Awesome, thanks @l-suarez. Let me know if/when this has been approved by the UIUX team (and whoever else is involved) and I'll update the PR accordingly 😄

Nino has given us a thumbs up, so good to go @PhilippeOberti 👍 thanks for working on this!

PhilippeOberti commented:
> > > Thanks for all the context @PhilippeOberti
> > > It looks like we added that ad-hoc icon more than 3 years ago as a solution to inform users about the type of data view. We already tell users if it's temporary, managed or both when we expand the dropdown to select a data view, and we do so adding a label component. I'd recommend doing the same for when it's selected instead of adding icons. This way is clearer, it'll make the select component longer, but it will be clearer. cc/ @ninoslavmiskovic
> > 
> > 
> > Awesome, thanks @l-suarez. Let me know if/when this has been approved by the UIUX team (and whoever else is involved) and I'll update the PR accordingly 😄
> 
> Nino has given us a thumbs up, so good to go @PhilippeOberti 👍 thanks for working on this!

The code has been updated, and I also updated the PR description to reflect the new changes. I decided to combine this PR with the [other one](https://github.com/elastic/kibana/pull/236195) (which I closed) as the overlap was too high now that we show the labels everywhere. 😄

PhilippeOberti committed:
fix FTR tests

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/345412)
* Commit: 3da06b9688e0092b3545f28c8bbcce7411a139f9

### Failed CI Steps
* [Scout: [ observability / apm ] plugin](https://buildkite.com/elastic/kibana-pull-request/builds/345412#0199979d-0659-46a1-839b-74db7de59bf0)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/345412#0199979d-0659-46a1-839b-74db7de59bf0) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/345412/jobs/0199979d-0659-46a1-839b-74db7de59bf0/artifacts/019997a6-4084-4870-b72e-77f7d809040b) Scout: [ observability / apm ] plugin / serverless-oblt - Service map - shows empty state when there is no data

### History
* :broken_heart: [Build #345276](https://buildkite.com/elastic/kibana-pull-request/builds/345276) failed 6dd4fb2eb79bca39f67268f1dc4cd94adbc3e649

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/345412","number":345412,"commit":"3da06b9688e0092b3545f28c8bbcce7411a139f9"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/345276","number":345276,"commit":"6dd4fb2eb79bca39f67268f1dc4cd94adbc3e649"}],"number":345412}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/346045)
* Commit: a5ae2acda4787f7772aead251ba31d34a6664dbc

### Failed CI Steps
* [Scout: [ platform / discover_enhanced ] plugin](https://buildkite.com/elastic/kibana-pull-request/builds/346045#01999c91-3fb3-4864-b364-d300f3d780c3)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/346045#01999c91-3fb3-4864-b364-d300f3d780c3) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/346045/jobs/01999c91-3fb3-4864-b364-d300f3d780c3/artifacts/01999ca8-b4b5-460c-b870-3c20b3b7fda4) Scout: [ platform / discover_enhanced ] plugin / serverless-oblt - Discover app - saved searches - should unselect saved search when navigating to a 'new'
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/346045#01999c91-3fb3-4864-b364-d300f3d780c3) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/346045/jobs/01999c91-3fb3-4864-b364-d300f3d780c3/artifacts/01999ca8-b4b5-4032-aefa-1e942fac2813) Scout: [ platform / discover_enhanced ] plugin / serverless-search - Discover app - saved searches - should unselect saved search when navigating to a 'new'
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/346045#01999c91-3fb3-4864-b364-d300f3d780c3) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/346045/jobs/01999c91-3fb3-4864-b364-d300f3d780c3/artifacts/01999ca8-b4b5-4c95-ae9e-cc1c94cbca56) Scout: [ platform / discover_enhanced ] plugin / stateful - Discover app - saved searches - should unselect saved search when navigating to a 'new'

### History
* :broken_heart: [Build #345918](https://buildkite.com/elastic/kibana-pull-request/builds/345918) failed 6098f49a61bf3452de8e8e26e5c26b2e2cd26d98
* :broken_heart: [Build #345493](https://buildkite.com/elastic/kibana-pull-request/builds/345493) failed c1d5f0a9e2ab162cf11ccd5b43636e7248e1d63f
* :broken_heart: [Build #345412](https://buildkite.com/elastic/kibana-pull-request/builds/345412) failed 3da06b9688e0092b3545f28c8bbcce7411a139f9
* :broken_heart: [Build #345276](https://buildkite.com/elastic/kibana-pull-request/builds/345276) failed 6dd4fb2eb79bca39f67268f1dc4cd94adbc3e649

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/346045","number":346045,"commit":"a5ae2acda4787f7772aead251ba31d34a6664dbc"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/345918","number":345918,"commit":"6098f49a61bf3452de8e8e26e5c26b2e2cd26d98"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/345493","number":345493,"commit":"c1d5f0a9e2ab162cf11ccd5b43636e7248e1d63f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/345412","number":345412,"commit":"3da06b9688e0092b3545f28c8bbcce7411a139f9"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/345276","number":345276,"commit":"6dd4fb2eb79bca39f67268f1dc4cd94adbc3e649"}],"number":346045}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/346467)
* Commit: db058c46083de7bed84b73f6d34441ace7b145f1

### Failed CI Steps
* [Scout: [ platform / discover_enhanced ] plugin](https://buildkite.com/elastic/kibana-pull-request/builds/346467#0199a169-2377-4e41-850f-bf0a0b3aa162)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/346467#0199a169-2377-4e41-850f-bf0a0b3aa162) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/346467/jobs/0199a169-2377-4e41-850f-bf0a0b3aa162/artifacts/0199a181-e186-4895-84ea-1df1d8d40a4b) Scout: [ platform / discover_enhanced ] plugin / serverless-oblt - Discover app - saved searches - should unselect saved search when navigating to a 'new'
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/346467#0199a169-2377-4e41-850f-bf0a0b3aa162) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/346467/jobs/0199a169-2377-4e41-850f-bf0a0b3aa162/artifacts/0199a181-e186-4a61-9c3f-b5a36d487bcf) Scout: [ platform / discover_enhanced ] plugin / serverless-search - Discover app - saved searches - should unselect saved search when navigating to a 'new'
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/346467#0199a169-2377-4e41-850f-bf0a0b3aa162) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/346467/jobs/0199a169-2377-4e41-850f-bf0a0b3aa162/artifacts/0199a181-e185-4627-a5a7-51aca174d5dc) Scout: [ platform / discover_enhanced ] plugin / stateful - Discover app - saved searches - should unselect saved search when navigating to a 'new'

### History
* :broken_heart: [Build #346045](https://buildkite.com/elastic/kibana-pull-request/builds/346045) failed a5ae2acda4787f7772aead251ba31d34a6664dbc
* :broken_heart: [Build #345918](https://buildkite.com/elastic/kibana-pull-request/builds/345918) failed 6098f49a61bf3452de8e8e26e5c26b2e2cd26d98
* :broken_heart: [Build #345493](https://buildkite.com/elastic/kibana-pull-request/builds/345493) failed c1d5f0a9e2ab162cf11ccd5b43636e7248e1d63f
* :broken_heart: [Build #345412](https://buildkite.com/elastic/kibana-pull-request/builds/345412) failed 3da06b9688e0092b3545f28c8bbcce7411a139f9
* :broken_heart: [Build #345276](https://buildkite.com/elastic/kibana-pull-request/builds/345276) failed 6dd4fb2eb79bca39f67268f1dc4cd94adbc3e649

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/346467","number":346467,"commit":"db058c46083de7bed84b73f6d34441ace7b145f1"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/346045","number":346045,"commit":"a5ae2acda4787f7772aead251ba31d34a6664dbc"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/345918","number":345918,"commit":"6098f49a61bf3452de8e8e26e5c26b2e2cd26d98"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/345493","number":345493,"commit":"c1d5f0a9e2ab162cf11ccd5b43636e7248e1d63f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/345412","number":345412,"commit":"3da06b9688e0092b3545f28c8bbcce7411a139f9"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/345276","number":345276,"commit":"6dd4fb2eb79bca39f67268f1dc4cd94adbc3e649"}],"number":346467}
buildkite-pr-comment-->

PhilippeOberti committed:
fix Scout test

elasticmachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/346497)
* Commit: e973a66560ca64e22cd3f41592b68adaba7beb02

### Failed CI Steps
* [FTR Configs #54](https://buildkite.com/elastic/kibana-pull-request/builds/346497#0199a1ad-98e7-4356-8ed9-46beada6687a)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/9dfdff1b3c7cb1585ad4761440b110d5c65f1cc8) | [after](https://github.com/elastic/kibana/commit/e973a66560ca64e22cd3f41592b68adaba7beb02) | diff |
| --- | --- | --- | --- |
| `unifiedSearch` | 386.5KB | 386.5KB | +31.0B |



### History
* :broken_heart: [Build #346467](https://buildkite.com/elastic/kibana-pull-request/builds/346467) failed db058c46083de7bed84b73f6d34441ace7b145f1
* :broken_heart: [Build #346045](https://buildkite.com/elastic/kibana-pull-request/builds/346045) failed a5ae2acda4787f7772aead251ba31d34a6664dbc
* :broken_heart: [Build #345918](https://buildkite.com/elastic/kibana-pull-request/builds/345918) failed 6098f49a61bf3452de8e8e26e5c26b2e2cd26d98
* :broken_heart: [Build #345493](https://buildkite.com/elastic/kibana-pull-request/builds/345493) failed c1d5f0a9e2ab162cf11ccd5b43636e7248e1d63f
* :broken_heart: [Build #345412](https://buildkite.com/elastic/kibana-pull-request/builds/345412) failed 3da06b9688e0092b3545f28c8bbcce7411a139f9
* :broken_heart: [Build #345276](https://buildkite.com/elastic/kibana-pull-request/builds/345276) failed 6dd4fb2eb79bca39f67268f1dc4cd94adbc3e649

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/346497","number":346497,"commit":"e973a66560ca64e22cd3f41592b68adaba7beb02"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/346467","number":346467,"commit":"db058c46083de7bed84b73f6d34441ace7b145f1"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/346045","number":346045,"commit":"a5ae2acda4787f7772aead251ba31d34a6664dbc"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/345918","number":345918,"commit":"6098f49a61bf3452de8e8e26e5c26b2e2cd26d98"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/345493","number":345493,"commit":"c1d5f0a9e2ab162cf11ccd5b43636e7248e1d63f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/345412","number":345412,"commit":"3da06b9688e0092b3545f28c8bbcce7411a139f9"}],"number":346497}
buildkite-pr-comment-->

pheyos reviewed (approved):
Scout test `saved_searches.spec.ts` changes LGTM

jughosta commented on src/platform/plugins/shared/unified_search/public/dataview_picker/change_dataview.tsx:
```diff
@@ -136,13 +141,12 @@ export function ChangeDataView({
-            </EuiFlexItem>
-          )}
-          <EuiFlexItem grow={false}>{trigger.label}</EuiFlexItem>
+          <EuiFlexItem data-test-subj={`${dataTestSubj}-label`} grow={false}>
+            {trigger.label}
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <DataViewLabels isAdhoc={isAdHocSelected} isManaged={isManagedSelected} name={label} />
+          </EuiFlexItem>
```
Can we please revert these changes? It clutters the main UI with long badges. We should prioritize giving more space to the search input instead. User can still find this information by opening the data views list. 

<img width="1261" height="335" alt="Screenshot 2025-10-02 at 10 21 08" src="https://github.com/user-attachments/assets/5fb1c1a5-bb05-44df-9778-21087f1853c2" />

PhilippeOberti replied:
hey @jughosta I'm confused, I had synced with @l-suarez and it [was suggested](https://github.com/elastic/kibana/pull/236187#issuecomment-3334478862) to make this change, which was then [approved](https://github.com/elastic/kibana/pull/236187#issuecomment-3347156255).

Can you guys sync on this? I've already spend a few hours trying to fix all the FTR tests which I was not familiar with and don't want to do any more changes until I get the green from everyone here.

The whole point of this PR was to improve the selected state. The previous code was only showing an icon but only for temporary selected data views. Managed data views didn't show anything, which was weird.

I actually originally had a [separate PR](https://github.com/elastic/kibana/pull/236195) that was taking care of the dropdown only, which I merged with this one once I got the green light to apply this fix in both places...

davismcphee replied:
@PhilippeOberti Sorry for the confusion. We've been pretty overloaded leading up to and since FF, and I haven't had the chance to look at much other than our high priority 9.2 work lately. I agree, let's hold off on any more changes here until we get a chance to sync on our side and figure things out.

I think what we're missing is more context for these changes. It makes sense to me that because we have an icon + badge for ad hoc data views, but only a badge for managed, we'd want to align them. But I don't really understand why we need both at once, especially since it's a lot of info to cram in. Are there cases where it's important for users to know a data view is both managed and ad hoc, or would it work to prioritize only managed indicators when applicable?

l-suarez replied:
While we wait for more context on the importance of showing this information on the closed dataview dropdown (considering this info is available when opening the list). I'd also like to challenge the need for an ad-hoc icon (does the user really know that this icon represents ad-hoc without opening the dropdown?). 

In any case, if we still see the need to make this info a first class citizen in Discover, I'd suggest adding a T or M to the dataview name, keeping consistency with how we show it on the opened data view list:

<img width="785" height="406" alt="image" src="https://github.com/user-attachments/assets/deab5e09-09ab-4f3c-a16c-01e223158f30" />

jughosta commented on src/platform/plugins/shared/unified_search/public/dataview_picker/data_view_labels.tsx:
```diff
@@ -0,0 +1,83 @@
+      ) : null}
+    </>
+  );
+});
```
Does not fit to one line:

<img width="458" height="150" alt="Screenshot 2025-10-02 at 10 15 09" src="https://github.com/user-attachments/assets/39f561a7-b6e3-4dda-9867-da5db2a67ed2" />

jughosta reviewed (changes requested):
Let's limit the changes only to the data views list in the dropdown.

markov00 reviewed (approved):
Code review only, LGTM

kertal commented:
We’ve decided not to move forward with adding the temporary and managed labels/badges in the data view picker in the collapsed state of the data view picker. The main reason is avoid sacrificing space of the displayed name for the label/badge we would add. We even considered to remove the current indicator icon for a temporary data view, but decided for now to keep it how it is. We might revisit this in the future.

For showing both `managed` and `temporary` we didn't identify a use case that would justify the loss of screen space by showing both. We think that a managed data view doesn't need to be labeled `temporary` additionally. Indicating the `temporary` state was added because when e.g. the Discover session was not saved, that the data view gets lost. This ain't the case for data views that are managed and temporary. 

Thank you for your effort, Philippe, and apologies that we didn’t align earlier, which resulted in some unnecessary work on your side.
